driver.go

```diff
@@ -507,6 +507,7 @@ func (d *Driver) StartTask(cfg *drivers.TaskConfig) (*drivers.TaskHandle, *drive
 				return nil, nil, fmt.Errorf("failed to start task, unable to pull image %s : %v", imageName, err)
 			}
 		}
+		createOpts.Image = imageName
```

Comment:
@towe75 I added this to fix an error with the digest. I understand that the issue is that podman treats differently images with tags and digests. Whereas tagged images are considered shortnames if there is not a fully-qualified reference, images with digests are not treated like that: https://github.com/containers/image/blob/v5.10.5/pkg/shortnames/shortnames.go#L17

Also,  the fact that `parseImage` uses `dockerref.ParseNormalizedNamed` deviates from standard podman behavior. It will insert `docker.io` by default if a domain is missing, completely ignoring podman shortname behavior. However, I think fixing that is out of the scope if this PR and deserves a separate discussion.

Reply:
Yes, IC and in fact i proposed this change in my first reply to your email: 

> Also the actual createOpts struct does not rely on the parsed and formatted name but still uses the "raw" driverConfig.Image.

It's not so simple to fix it in a clean way until there is better tooling/library-support for podman available.

Reply:
OK, perfect. Sorry I did not realize about that until it failed. I will open an issue to track this, might have the time to look into it in the following weeks.